k-distribution/tests/regression-new/checks/concrete.k

```diff
@@ -0,0 +1,26 @@
+module CONCRETE
+  imports INT
+
+  rule 0 +Int X => X [symbolic, simplification]
+  rule 1 +Int X => X +Int 1 [concrete, simplification]
+  rule 0 *Int X => 0 [symbolic]
+  rule 1 *Int X => X [concrete]
+
+  syntax Int ::= foo(Int) [function]
+  rule foo(0) => 0 [symbolic, concrete]
+
+  syntax Int ::= bar(Int) [function]
+  rule bar(0) => 0 [symbolic]
+
+  syntax Int ::= bar2(Int) [function]
+  rule bar2(0) => 0 [symbolic]
+  rule bar2(1) => 1
+
+  syntax Int ::= baz(Int) [function]
+  rule baz(0) => 0 [concrete]
+
+  syntax Int ::= baz2(Int) [function]
+  rule baz2(0) => 0 [concrete]
+  rule baz2(1) => 1
+
+endmodule
```

Comment:
Just to make sure, can you add a test that uses `concrete(X)` or `symbolic(X,Y)` as well? Because that's another form that these show up in.